src/kontrol/foundry.py

```diff
@@ -309,9 +309,14 @@ def get_test_id(self, test: str, version: int | None) -> str:
         if len(matching_proof_ids) == 0:
             raise ValueError(f'Found no matching proofs for {test}:{version}.')
         if len(matching_proof_ids) > 1:
-            raise ValueError(
-                f'Found {len(matching_proof_ids)} matching proofs for {test}:{matching_proof_ids}. Use the --version flag to choose one.'
-            )
+            if version == None:
```

Comment:
```suggestion
            if version is None:
```
I think it's recommended to use `is None` instead of `== None`, e.g., https://discuss.python.org/t/why-is-it-better-to-write-if-a-is-none-instead-of-if-a-none/42203